src/pages/PackMuleCrafter.vue

```diff
@@ -65,92 +66,122 @@
+                {{ recipe }}
               </div>
-              <div class="border-top border-bottom col-sm-10">{{ recipe }}</div>
             </div>
           </div>
+        </div>
+        <div class="col-sm-4" id="materials">
           <h4>Material List</h4>
           <div
             class="container"
             id="materialList"
-            style="height: 200px; overflow: auto"
+            style="height: 300px; overflow: auto"
           >
             <div
               class="row"
               v-for="(material, index) in recommended.materials"
               :key="`material-${index}`"
             >
-              <div class="border-top border-bottom col-sm-2" />
-              <div class="border-top border-bottom col-sm-10">
+              <div class="border-top border-bottom col-sm-3">
+                <GameAsset
+                  :height="72"
+                  :image="Assets.MaterialImage(material.replace(/[0-9]+ /g, '').replace(/ /g, '_'))"
+                  :title="material.replace(/[0-9]+ /g, '')"
+                >
+                  <template #tooltip>
+                    <div v-html="material.replace(/[0-9]+ /g, '')"></div>
+                  </template>
+                </GameAsset>
+              </div>
+              <div class="border-top border-bottom col-sm-9">
                 {{ material }}
               </div>
             </div>
           </div>
         </div>
-        <div class="col-sm-8" id="inventoryList">
-          <h4>Inventory Capacity Setups</h4>
-          <div class="container" style="height: 475px; overflow: auto">
-            <div
-              class="row"
-              v-for="(inventory, index) in recommended.caps"
-              :key="`inventory-${index}`"
-            >
-              <div class="border-top border-bottom col-sm">
-                <h5>Slots: {{ inventory.slots }}</h5>
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Material:<br />{{ inventory.material }}
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Mining:<br />{{ inventory.mining }}
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Fish:<br />{{ inventory.fish }}
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Food:<br />{{ inventory.food }}
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Chopping:<br />{{ inventory.choppin }}
-              </div>
-              <div class="border-top border-bottom col-sm">
-                Bug:<br />{{ inventory.bug }}
-              </div>
+      </div>
+      
+      <div class="row">
+        <h4>Inventory Capacity Setups</h4>
+        <div class="container" style="height: 475px; overflow: auto">
+          <div
+            class="row"
+            v-for="(inventory, index) in recommended.caps"
```

Comment:
You probably addressed this in the other PR or were planning to, but this can infer the slots from `curCharacter.bagSlots` IIRC

Reply:
Added